tools/serverpod_cli/test/generator/dart/server_code_generator/class_copy_with_test.dart

```diff
@@ -343,8 +344,8 @@ void main() {
     test('then the clone method is called on field when copying the object.',
         () {
       var sourceCode = copyWithMethod?.body.toSource();
-      expect(
-          sourceCode, '{return Example(names: names ?? this.names.clone());}');
+      expect(sourceCode,
+          '{return Example(names: names ?? this.names.map((e0) => _i1.$cloneTrivialFunctionName(e0)).toList());}');
```

Comment:
Suggestion: Align description with the new expect.

We no longer simply call clone on the object. Please align the description with the new expect of the test.